BibTeX.js

```diff
@@ -1173,6 +1173,20 @@ function doExport() {
 		// determine type
 		var type = zotero2bibtexTypeMap[item.itemType];
 		if (typeof(type) == "function") { type = type(item); }
+
+		// The thesis Zotero type doesn't nicely map onto a BibTeX type,
+		// as BibTeX uses both @mastersthesis and @phdthesis. We thus have
+		// to tell them apart by the "Type" field as well as by the item type.
+		if (type == "phdthesis") {
+			// In practice, we just want to separate out masters theses,
+			// and will assume everything else maps to @phdthesis. Better to
+			// err on the side of caution.
+			if (item["type"] && item["type"].toLowerCase() == "masters") {
```

Comment:
This looks good to me too.  If we're going to be very specific, we should ensure BibTeX imports gets the proper item.type populated by whatever we're trying to prescribe.

I have no idea what people tend to use for the "Type" field in Zotero.  Is there a way to survey usage to make sure that whatever pattern we select is both sane and comprehensive?